neurom/fst/_neuronfunc.py

```diff
@@ -140,6 +140,57 @@ def _elevation(section, soma):
             for s in n.neurites if neurite_filter(s)]
 
 
+def trunk_vectors(nrn, neurite_type=NeuriteType.all):
+    '''Calculates the vectors between all the trunks of the neuron
+    and the soma center.
+    '''
+    neurite_filter = is_type(neurite_type)
+    nrns = neuron_population(nrn)
+
+    return np.array([morphmath.vector(s.root_node.points[0], n.soma.center)
+                     for n in nrns
+                     for s in n.neurites if neurite_filter(s)])
+
+
+def trunk_angles(nrn, neurite_type=NeuriteType.all):
+    '''Calculates the angles between all the trunks of the neuron.
+    The angles are defined on the x-y plane and the trees
+    are sorted from the y axis and anticlock-wise.
+    '''
+    vectors = trunk_vectors(nrn, neurite_type=neurite_type)
+
+    def _angle_between(p1, p2):
```

Comment:
_angle_between should be defined in morphmath so it can be reused

Reply:
OK will move this